.gitignore

```diff
@@ -5,3 +5,6 @@ __pycache__
 foo.png
 foo*.svg
 .DS_Store
+
+js/node_modules
+js/dist
```

Comment:
I'd prefer these lines in ./js/.gitignore

Reply:
Done